packages/cli/test/integration/generators/app.integration.js

```diff
@@ -295,7 +295,12 @@ describe('app-generator with tilde project path', () => {
     // eslint-disable-next-line no-invalid-this
     this.timeout(30 * 1000);
 
-    process.chdir(sandbox);
+    // If the test runs outside $home directory
+    if (process.env.CI && !process.env.DEBUG) {
```

Comment:
I am little bit concerned about duplicating the logic deciding whether to use `sandbox` or `rootDir`, because if we change the original condition [on line 266](https://github.com/strongloop/loopback-next/blob/9aba52e59c80830dd0871c4a087b90ec2aa21db2/packages/cli/test/integration/generators/app.integration.js#L266), then how are we going to ensure this copy is updated as well?

If we want to change to a different directory, then let's introduce a new variable set to true/false depending on whether the condition on line 266 was triggered or not.

Having said that, I am not sure if changing the directory to `rootDir` is the right thing to do, see my comment below.

Reply:
> If we want to change to a different directory, then let's introduce a new variable set to true/false depending on whether the condition on line 266 was triggered or not.

New variable utilised.